client/src/App.js

```diff
@@ -19,13 +20,19 @@ class App extends Component {
   }
 
   componentDidMount() {
-    axios.get(`/api/places`).then(({ data }) => {
-      this.setState({ listOfPlaces: data })
-    })
+    axios
+      .get(`/api/places`)
+      .then(({ data }) => {
+        this.setState({ listOfPlaces: data })
+      })
+      .catch(() => this.props.history.push('/error500'))
 
-    axios.get('/api/guides').then(({ data }) => {
-      this.setState({ listOfGuides: data })
-    })
+    axios
+      .get('/api/guides')
+      .then(({ data }) => {
+        this.setState({ listOfGuides: data })
+      })
+      .catch(() => this.props.history.push('/error500'))
```

Comment:
Generally you want to check for error status first. But we don't have other potential errors like 401 or 400 here.

Reply:
No changes are required here.